react/ProductDetails.js

```diff
@@ -66,16 +76,39 @@ class ProductDetails extends Component {
             <div className="pv2">
               <hr className="b--black-10" size="0" />
             </div>
+            <div>
+              <div className="f7">
+                <FormattedMessage id="sku-label" />
+              </div>
+              <SKUSelector
+                skuItems={product.items}
+                onSKUSelected={this.handleSkuChange}
+              />
+            </div>
             <div className="pv2">
+              <hr className="b--black-10" size="0" />
+            </div>
+            <div className="flex w-100 pv2">
+              <div className="flex pr2">
+                <BuyButton
+                  seller={sellerId}
+                  skuId={selectedItem.itemId}
+                >
+                  <FormattedMessage id="button-label" />
+                </BuyButton>
+              </div>
+            </div>
+            <div className="pv4">
               <ShippingSimulator skuId={selectedItem.itemId} seller={sellerId} country="BRA" />
             </div>
-            <div className="pv2">
-              <BuyButton
-                seller={sellerId}
-                skuId={selectedItem.itemId}
-              >
-                <FormattedMessage id="button-label" />
-              </BuyButton>
+            <div className="flex w-100 pv2">
+              <div className="pv2 pr3 f6">
+                <FormattedMessage id="share-label" />:
+              </div>
+              <Share 
+                options={{'size': '25'}} 
+                social={{'Facebook': true, 'WhatsApp': true, 'Twitter': true }} 
```

Comment:
These are the default values. Will you pass them anyway?

Reply:
Nops.